Assets/Scripts/LoadSceneProgress.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class LoadSceneProgress : MonoBehaviour
+{
+    [SerializeField] private Image loadingProgressFillImage;
+    
+    private AsyncOperation _loadOperation; 
+    
+    private void Start()
+    {
+        _loadOperation = SceneManager.LoadSceneAsync(LoadingData.SceneToLoad);
+    }
+
+    private void Update()
+    {
+        DisplayProgress();
+    }
+
+    private void DisplayProgress()
+    {
+        loadingProgressFillImage.fillAmount = Mathf.Clamp01(_loadOperation.progress / 0.9f);
```

Comment:
why .9?

Reply:
because the loading phase is only between 0.0 - 0.9